lib/_tools.scss

```diff
@@ -76,14 +75,10 @@
         -webkit-flex-flow: row wrap;
         -ms-flex-flow: row wrap;
         flex-flow: row wrap;
-        -webkit-flex: 1 1 auto; // [2]
-        -ms-flex: 1 1 auto; // [2]
-        flex: 1 1 auto; // [2]
-
-        margin-right: 0; // [3]
+        margin-right: 0; // [2]
```

Comment:
Will it be possible to make this work for `rtl` by interpolating the properties? `#{$margin-right}`, or is that only possible at the Grandstand level?